### PR TITLE
Appends vector tile API key to tile requests using HttpHandler

### PR DESCRIPTION
**Pull request by ecgreb**

### CODE REVIEW

File: app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt [hunk 1]
```diff
@@ -73,6 +74,8 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
         @Inject set
     var settings: AppSettings? = null
         @Inject set
+    var tileHttpHandler: TileHttpHandler? = null
+        @Inject set
```
Comment (tallytalwar) [inline app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt]: is this some Kotlin thingy?

**Comment by tallytalwar** [comment]
Looks good to me.

File: app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt (hunk 1 above)
Comment (ecgreb) [inline app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt]: `@Inject set` tells Dagger to inject this dependency via its Kotlin setter.

**Comment by blair1618** [comment]
This looks fine to me - can we merge it so that the automated builds start showing tiles again? :)

I will also add a default HTTP handler like you mentioned yesterday, that won't require any changes on the eraser-map side.

**Pull request merged**